pkg/apiserver/domain/service/user.go

```diff
@@ -415,3 +416,13 @@ func compareHashWithPassword(hash, password string) error {
 	}
 	return err
 }
+
+// NewTestUserService create the user service instance for testing
+func NewTestUserService(ds datastore.DataStore, c client.Client) UserService {
```

Comment:
Similar question, is this for test only?

Reply:
Yes, only for tests.